components/service/glean/src/main/java/mozilla/components/service/glean/ping/BaselinePing.kt

```diff
@@ -13,7 +13,7 @@ import java.util.Locale
 /**
  * BaselinePing facilitates setup and initialization of baseline pings.
  */
-internal class BaselinePing(applicationContext: Context) {
+internal class BaselinePing(@Suppress("UNUSED_PARAMETER") applicationContext: Context) {
```

Comment:
Let's remove `applicationContext` if we are not using it

Reply:
Ok.  Easy enough.